distribution/docs/src/main/resources/content/_architectures/url-resource-reader.adoc

```diff
@@ -31,6 +31,25 @@ Configure the URL Resource Reader from the ${admin-console}.
 
 See <<ddf.catalog.resource.impl.URLResourceReader,URL Resource Reader configurations>> for all possible configurations.
 
+=== Configuring Permissions for the URL Resource Reader
```

Comment:
I followed the model for the CDM here. Some of it is redundant with that documentation.